clang/include/clang/Basic/TargetInfo.h

```diff
@@ -358,7 +358,10 @@ class TargetInfo : public TransferrableTargetInfo,
     //    void *__saved_reg_area_end_pointer;
     //    void *__overflow_area_pointer;
     //} va_list;
-    HexagonBuiltinVaList
+    HexagonBuiltinVaList,
+
+    // Tensilica Xtensa
+    XtensaABIBuiltinVaList
```

Comment:
It would be helpful to stick a brief definition of the structure in a comment here, for comparison,

Reply:
fixed